tests/beta/test_default_tensor.py

```diff
@@ -331,6 +338,304 @@ def test_hermitian_exceptions(self):
             hermitian(H2)
  
 
+class TestDefaultTensorNetwork:
+    """Tests of the basic tensor network functionality of default.tensor plugin."""
+
+    def clear_network(self, tensornet_device_2_wires):
+        """Tests that the _clear_network method clears the relevant bookkeeping data."""
+
+        dev = tensornet_device_2_wires
+
+        @qml.qnode(dev)
+        def circuit():
+            qml.PauliX(wires=0)
+            qml.PauliY(wires=1)
+            return qml.expval(qml.PauliZ(0)), qml.sample(qml.PauliY(1))
+
+        circuit()
+        dev.reset()
```

Comment:
Just a quick question: would it make sense to only have a call on `dev.clear_network` here? It seems that the next test will anyways explore resetting further. (However, it is also fine to leave it as is, as `dev.reset` places the call well)

Reply:
Good catch, just looks like an oversight (this test never ran anyway because the name also has a typo). Fixed